ui/src/App.test.tsx

```diff
@@ -18,6 +19,7 @@ describe("App", () => {
 
   afterAll(() => {
     jest.useRealTimers();
+    jest.clearAllMocks();
```

Comment:
can remove?

Reply:
I'd rather clear the mocks even if it's not clear we need it quite yet. I'm fine leaving this in for now

Reply:
I think we should remove it if it doesn't do anything